unitypack/type.py

```diff
@@ -52,7 +52,8 @@ def load_old(self, buf):
 	def load_blob(self, buf):
 		num_nodes = buf.read_uint()
 		self.buffer_bytes = buf.read_uint()
-		node_data = BytesIO(buf.read(24 * num_nodes))
+		node_bytes = 32 if self.format >= 20 else 24
```

Comment:
Does anybody know what version this format change occurred in? My use cases are in 2018.1 (which is asset format 17) and 2019.2 (asset format 20). The correct value could be anywhere in there. I put 20 here for safety.

Reply:
I've only ever worked in reverse; can you download old versions of the Unity toolkit and make stub projects with empty-ish bundles to test?

Reply:
Unity 2018.4 is asset format 17 (24-byte nodes), and 2019.1 is format 19 (32-byte nodes). There weren't any minor releases between those, so I don't know where format 18 could be. Hopefully they're not changing asset formats in a patch update.

Reply:
Sounds good. I would merge the previous commit with this one entirely to avoid the incorrect intermediate commit; and possibly even the subsequent commit.